content/en/examples/deployments/deployment-with-immutable-configmap-as-volume.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: immutable-configmap-volume
+  labels:
+    app.kubernetes.io/name: immutable-configmap-volume
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: immutable-configmap-volume
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: immutable-configmap-volume
+    spec:
+      containers:
+        - name: alpine
+          image: alpine:3
+          command:
+            - /bin/sh
+            - -c
+            - while true; do echo "$(date) My preferred season is $(cat /etc/config/season)";
```

```suggestion
            - while true; do echo "$(date) The current season is $(cat /etc/config/season)";
```
or, if following other advice:
```shell
            - while true; do echo "$(date) The name of the company is $(cat /etc/config/company_name)";
```

---

I have aligned this with the example of the company names.